### PR TITLE
Remove extra import

### PR DESCRIPTION
**Pull request by yash-garg**

### CODE REVIEW

File: uupdump.py [hunk 1]
```diff
@@ -126,4 +126,4 @@
                 resp = requests.post(URL, headers=headers, data=data)
```
Comment (baalajimaestro) [inline uupdump.py:126]: Requests is used, re isnt, agreed. But this code badly needs a rewrite. Ill do it sometime later. I wanted to make it working than a clean code.

**Review by baalajimaestro — CHANGES_REQUESTED** [review]

File: uupdump.py (hunk 1 above)
Comment (baalajimaestro) [reply uupdump.py:126]: If I were to rate my code for quality I would give it a -1/10

**Review by baalajimaestro — APPROVED** [review]

File: uupdump.py (hunk 1 above)
Comment (yash-garg) [reply uupdump.py:126]: lol ok xD, i mean u already imported it on first line

Comment (baalajimaestro) [reply uupdump.py:126]: I saw that. Thats why I approved and merged .-.

**Pull request merged**